pkg/backend/aws/ecs.go

```diff
@@ -14,6 +14,68 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func (b *Backend) DescribeService(ctx context.Context, serviceName *string) (*ecs.Service, error) {
+	clusterARN := b.integrationSecret.ClusterArn
+	out, err := b.ecsclient.DescribeServicesWithContext(ctx, &ecs.DescribeServicesInput{
+		Cluster:  &clusterARN,
+		Services: []*string{serviceName},
+	})
+	if err != nil {
+		return nil, errors.Wrap(err, "cannot describe an ECS service")
+	}
+	if len(out.Services) == 0 {
+		return nil, errors.Wrap(err, "ECS service was not found")
```

Comment:
```suggestion
		return nil, errors.New("ECS service was not found")
```
since error is nil here, wrapping it will actually also return nil